src/context.tsx

```diff
@@ -1,8 +1,19 @@
 import React, { createContext, Dispatch, useReducer } from "react";
-import { AggregationFactor, Filters, FilterType, LanguageType, PageState, State } from "./types";
-import Translate from "./utils/translate/translateService";
+import {
+  AggregationFactor,
+  Filters,
+  FilterType,
+  LanguageType,
+  PageState,
+  State,
+} from "./types";
 
-export const AppContext = createContext({} as [State, Dispatch<Record<string, any>>]);
+export const AppContext = createContext(
+  {} as [State, Dispatch<Record<string, any>>]
+);
+
+const minDate = new Date(2019, 1, 1, 1);
```

Comment:
Let's make a small ticket to make this min date reflective of our database state (earliest date in our database). Not a huge priority imo but would be nice @annoyingpr

Reply:
Also nit: what do we think about renaming these consts to `initialMaxDate` and `initialMinDate` to make sure these are the initial bounds on the calendar